apps/s_client.c

```diff
@@ -3827,7 +3827,7 @@ static int user_data_process(struct user_data_st *user_data, size_t *len,
         char *cmd_start = buf_start;
 
         cmd_start[outlen] = '\0';
-        do {
+        for (;;) {
             cmd_start = strstr(cmd_start, "{");
```

Comment:
When you're at it, you could also replace these single character strstr() calls with strchr() calls.